versions/run.noaacloud.ver

```diff
@@ -0,0 +1,11 @@
+export stack_intel_ver=2021.3.0
+export stack_impi_ver=2021.3.0
+export spack_env=gsi-addon-env
+
+export gempak_ver=7.4.2
+
+source "${HOMEgfs:-}/versions/run.spack.ver"
+export spack_mod_path="/contrib/spack-stack/spack-stack-${spack_stack_ver}/envs/gsi-addon-env/install/modulefiles/Core"
+
+export ncl_ver=6.6.2
```

Comment:
Tried remove it, will cause compare time error.
I used to be NCL developer, and do not think we need NCL for forecast or atmos-product,
but probably some cmake requirement people put in.

Reply:
Where did you encounter compare time error?

Reply:
Without set NCL, we got error at stage_ic:

Begin stage_ic.sh at Fri Jul  5 16:26:28 UTC 2024
+ stage_ic.sh[6]: . /contrib/Wei.Huang/src/global-workflow-cloud/ush/load_fv3gfs_modules.sh
++ load_fv3gfs_modules.sh[4]: [[ NO == \N\O ]]
++ load_fv3gfs_modules.sh[5]: echo 'Loading modules quietly...'
Loading modules quietly...
++ load_fv3gfs_modules.sh[6]: set +x
Lmod has detected the following error: The following module(s) are unknown:
"ncl/None"

Please check the spelling or version number. Also try "module spider ..."
It is also possible your cache file is out-of-date; it may help to try:
  $ module --ignore_cache load "ncl/None"

Also make sure that all modulefiles written in TCL start with the string
#%Module

Executing this command requires loading "ncl/None" which failed while
processing the following module(s):

    Module fullname        Module Filename
    ---------------        ---------------
    module_base.noaacloud  /contrib/Wei.Huang/src/global-workflow-cloud/modulefiles/module_base.noaacloud.lua


End stage_ic.sh at 16:26:40 with error code 1 (time elapsed: 00:00:12)

Reply:
removed ncl-version.